Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ListViewExtensions/ListViewExtensionsPage.xaml.cs

```diff
@@ -15,29 +16,71 @@ namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
 {
     public sealed partial class ListViewExtensionsPage : Page, IXamlRenderListener
     {
+        private ListView sampleListView;
+        private TextBlock indexInput;
+        private TextBlock itemPlacementInput;
+        private CheckBox disableAnimationInput;
+        private CheckBox scrollIfVisibileInput;
+        private TextBlock additionalHorizontalOffsetInput;
+        private TextBlock additionalVerticalOffsetInput;
+
         public ListViewExtensionsPage()
         {
             this.InitializeComponent();
+            Load();
         }
 
-        public ICommand SampleCommand => new DelegateCommand<PhotoDataItem>(OnExecuteSampleCommand);
+        public ICommand SampleCommand => new DelegateCommand<string>(OnExecuteSampleCommand);
 
-        public async void OnXamlRendered(FrameworkElement control)
+        public void OnXamlRendered(FrameworkElement control)
         {
-            var sampleListView = control.FindChildByName("SampleListView") as ListView;
+            sampleListView = control.FindChildByName("SampleListView") as ListView;
+            indexInput = control.FindChildByName("IndexInput") as TextBlock;
+            itemPlacementInput = control.FindChildByName("ItemPlacementInput") as TextBlock;
+            disableAnimationInput = control.FindChildByName("DisableAnimationInput") as CheckBox;
+            scrollIfVisibileInput = control.FindChildByName("ScrollIfVisibileInput") as CheckBox;
+            additionalHorizontalOffsetInput = control.FindChildByName("AdditionalHorizontalOffsetInput") as TextBlock;
+            additionalVerticalOffsetInput = control.FindChildByName("AdditionalVerticalOffsetInput") as TextBlock;
 
             if (sampleListView != null)
             {
-                sampleListView.ItemsSource = await new Data.PhotosDataSource().GetItemsAsync();
+                sampleListView.ItemsSource = GetOddEvenSource(201);
             }
 
             // Transfer Data Context so we can access SampleCommand
             control.DataContext = this;
```

Comment:
Setting `DataContext` breaks all sample app xaml property binding (for eg `@[Index:Slider:100:0-200]`). Is that an expected behaviour? Or should I create an issue? @michael-hawker Thoughts

Reply:
@Vijay-Nirmal sorry I missed your question here. Just getting to the backlog of PRs finally.

It's definitely not intended currently to support a custom `DataContext` with the property sliders. They're kind of meant for two independent scenarios of having the properties or just having a XAML example.

You can see this pretty clearly as the PropertyControl is casting the DataContext to a Sample object here:

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/30452cf0bbf627f12825cf50bbd31b0e526b9abe/Microsoft.Toolkit.Uwp.SampleApp/Controls/PropertyControl.xaml.cs#L48

Not sure how easy it'd be to change this or what the expectations of how it'd work would be. Trying to think how this example worked before vs. what we're trying to do now?

If you want to chat on a call quick about this sometime, let me know. It'd be great to finally get this PR merged. Are you on the [UWP Community Discord server](https://discord.com/invite/zBA5aCn)?

Reply:
> Trying to think how this example worked before vs. what we're trying to do now?

This worked before because at that time we didn't have `ListViewExtensions.Command`

I have joined UWP Community Discord Server

Reply:
@Vijay-Nirmal did my comment give you enough detail to resolve your issue in the sample or would you like to meet-up and discuss together? Feel free to give me a ping on the Discord and we can setup a session sometime if you'd like.

Reply:
@Vijay-Nirmal any updates? Did you find some time to re-work the DataContext conflict?

Reply:
I didn't find a way to fix the DataContext conflict

Reply:
@RosarioPulella after we get through the refactors this week (so maybe the week after or the next, if you're around) - want to take a look to see if we can help out here to get this over the line?

Reply:
@michael-hawker Sure, I can provide some help.

Reply:
@michael-hawker I made it so we don't have have to set the data context. I fallowed the pattern from the `InAppNotifation` sample.

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/63cbb4a51bdef59083ccfc86bbcba41f966d0027/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InAppNotification/InAppNotificationPage.xaml.cs#L185-L209

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/63cbb4a51bdef59083ccfc86bbcba41f966d0027/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InAppNotification/InAppNotificationPage.xaml#L13